<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>uiSR_RFRectifiers</class>
 <widget class="QMainWindow" name="uiSR_RFRectifiers">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>591</width>
    <height>331</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Storage Ring RF SSA Rectifiers</string>
  </property>
  <widget class="QWidget" name="centralWidget">
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>110</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>REC 00</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>110</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>REC 01</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Current</string>
   </property>
  </widget>
  <widget class="QSimpleShape" name="ledConnectionREC3">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>130</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="shape">
    <enum>QSimpleShape::circle</enum>
   </property>
   <property name="colour0" stdset="0">
    <color>
     <red>0</red>
     <green>255</green>
     <blue>0</blue>
    </color>
   </property>
   <property name="colour1" stdset="0">
    <color>
     <red>0</red>
     <green>100</green>
     <blue>0</blue>
    </color>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>DC Voltage</string>
   </property>
  </widget>
  <widget class="QEPushButton" name="btnSetAmpVoltage">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>240</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Set</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Power</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_17">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>PA - Voltage</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>110</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>REC 03</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="titleGroupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>570</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox {
	background-color: rgb(255, 255, 255);
    border: 1px solid black;
    border-radius: 5px;
    margin-top: 0.5em;
}

QGroupBox::title {
    subcontrol-origin: margin;
    left: 10px;
    padding: 0 3px 0 3px;
}</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QPushButton" name="btnClose">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>20</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Close</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="titleGroupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>570</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox {
	background-color: rgb(255, 255, 255);
    border: 1px solid black;
    border-radius: 5px;
    margin-top: 0.5em;
}

QGroupBox::title {
    subcontrol-origin: margin;
    left: 10px;
    padding: 0 3px 0 3px;
}</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="title_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>550</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sans</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Storage Ring RF SSA $(ssa) Rectifiers</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QELineEdit" name="lineEditPAVoltage">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>240</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="variable" stdset="0">
    <string>SRC03-RF-SSA$(ssa)-REC00:setDcVoltage</string>
   </property>
   <property name="variableSubstitutions" stdset="0">
    <string/>
   </property>
  </widget>
  <widget class="QSimpleShape" name="ledConnectionREC1">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>130</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="shape">
    <enum>QSimpleShape::circle</enum>
   </property>
   <property name="colour0" stdset="0">
    <color>
     <red>0</red>
     <green>255</green>
     <blue>0</blue>
    </color>
   </property>
   <property name="colour1" stdset="0">
    <color>
     <red>0</red>
     <green>100</green>
     <blue>0</blue>
    </color>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>110</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>REC 04</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>110</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>REC 02</string>
   </property>
  </widget>
  <widget class="QSimpleShape" name="ledConnectionREC5">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>130</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="shape">
    <enum>QSimpleShape::circle</enum>
   </property>
   <property name="colour0" stdset="0">
    <color>
     <red>0</red>
     <green>255</green>
     <blue>0</blue>
    </color>
   </property>
   <property name="colour1" stdset="0">
    <color>
     <red>0</red>
     <green>100</green>
     <blue>0</blue>
    </color>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>80</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Amplifiers</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>80</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Pre-Amplifier</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>125</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Connection</string>
   </property>
  </widget>
  <widget class="QSimpleShape" name="ledConnectionREC4">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>130</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="shape">
    <enum>QSimpleShape::circle</enum>
   </property>
   <property name="colour0" stdset="0">
    <color>
     <red>0</red>
     <green>255</green>
     <blue>0</blue>
    </color>
   </property>
   <property name="colour1" stdset="0">
    <color>
     <red>0</red>
     <green>100</green>
     <blue>0</blue>
    </color>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>240</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Amplifiers Voltage</string>
   </property>
  </widget>
  <widget class="QSimpleShape" name="ledConnectionREC0">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>130</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="shape">
    <enum>QSimpleShape::circle</enum>
   </property>
   <property name="colour0" stdset="0">
    <color>
     <red>0</red>
     <green>255</green>
     <blue>0</blue>
    </color>
   </property>
   <property name="colour1" stdset="0">
    <color>
     <red>0</red>
     <green>100</green>
     <blue>0</blue>
    </color>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>110</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>REC 05</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>AC Voltage</string>
   </property>
  </widget>
  <widget class="QSimpleShape" name="ledConnectionREC2">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>130</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="shape">
    <enum>QSimpleShape::circle</enum>
   </property>
   <property name="colour0" stdset="0">
    <color>
     <red>0</red>
     <green>255</green>
     <blue>0</blue>
    </color>
   </property>
   <property name="colour1" stdset="0">
    <color>
     <red>0</red>
     <green>100</green>
     <blue>0</blue>
    </color>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditAmpVoltage">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>240</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QELabel" name="qelabel">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>150</y>
     <width>70</width>
     <height>20</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="variable" stdset="0">
    <string>SRC03-RF-SSA$(ssa)-REC$(rec):getDcVoltage</string>
   </property>
   <property name="variableSubstitutions" stdset="0">
    <string>rec=00</string>
   </property>
  </widget>
  <widget class="QELabel" name="qelabel_2">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>150</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="variable" stdset="0">
    <string>SRC03-RF-SSA$(ssa)-REC$(rec):getDcVoltage</string>
   </property>
   <property name="variableSubstitutions" stdset="0">
    <string>rec=01</string>
   </property>
  </widget>
  <widget class="QELabel" name="qelabel_3">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>150</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="variable" stdset="0">
    <string>SRC03-RF-SSA$(ssa)-REC$(rec):getDcVoltage</string>
   </property>
   <property name="variableSubstitutions" stdset="0">
    <string>rec=02</string>
   </property>
  </widget>
  <widget class="QELabel" name="qelabel_4">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>150</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="variable" stdset="0">
    <string>SRC03-RF-SSA$(ssa)-REC$(rec):getDcVoltage</string>
   </property>
   <property name="variableSubstitutions" stdset="0">
    <string>rec=03</string>
   </property>
  </widget>
  <widget class="QELabel" name="qelabel_5">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>150</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="variable" stdset="0">
    <string>SRC03-RF-SSA$(ssa)-REC$(rec):getDcVoltage</string>
   </property>
   <property name="variableSubstitutions" stdset="0">
    <string>rec=04</string>
   </property>
  </widget>
  <widget class="QELabel" name="qelabel_6">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>150</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="variable" stdset="0">
    <string>SRC03-RF-SSA$(ssa)-REC$(rec):getDcVoltage</string>
   </property>
   <property name="variableSubstitutions" stdset="0">
    <string>rec=05</string>
   </property>
  </widget>
  <widget class="QELabel" name="qelabel_7">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>170</y>
     <width>70</width>
     <height>20</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="variable" stdset="0">
    <string>SRC03-RF-SSA$(ssa)-REC$(rec):getAcVoltage</string>
   </property>
   <property name="variableSubstitutions" stdset="0">
    <string>rec=00</string>
   </property>
  </widget>
  <widget class="QELabel" name="qelabel_8">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>190</y>
     <width>70</width>
     <height>20</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="variable" stdset="0">
    <string>SRC03-RF-SSA$(ssa)-REC$(rec):getCurrent</string>
   </property>
   <property name="variableSubstitutions" stdset="0">
    <string>rec=00</string>
   </property>
  </widget>
  <widget class="QELabel" name="qelabel_9">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>210</y>
     <width>70</width>
     <height>20</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="variable" stdset="0">
    <string>SRC03-RF-SSA$(ssa)-REC$(rec):getPower</string>
   </property>
   <property name="variableSubstitutions" stdset="0">
    <string>rec=00</string>
   </property>
  </widget>
  <widget class="QELabel" name="qelabel_10">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>210</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="variable" stdset="0">
    <string>SRC03-RF-SSA$(ssa)-REC$(rec):getPower</string>
   </property>
   <property name="variableSubstitutions" stdset="0">
    <string>rec=01</string>
   </property>
  </widget>
  <widget class="QELabel" name="qelabel_11">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>190</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="variable" stdset="0">
    <string>SRC03-RF-SSA$(ssa)-REC$(rec):getCurrent</string>
   </property>
   <property name="variableSubstitutions" stdset="0">
    <string>rec=01</string>
   </property>
  </widget>
  <widget class="QELabel" name="qelabel_12">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>170</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="variable" stdset="0">
    <string>SRC03-RF-SSA$(ssa)-REC$(rec):getAcVoltage</string>
   </property>
   <property name="variableSubstitutions" stdset="0">
    <string>rec=01</string>
   </property>
  </widget>
  <widget class="QELabel" name="qelabel_13">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>210</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="variable" stdset="0">
    <string>SRC03-RF-SSA$(ssa)-REC$(rec):getPower</string>
   </property>
   <property name="variableSubstitutions" stdset="0">
    <string>rec=02</string>
   </property>
  </widget>
  <widget class="QELabel" name="qelabel_14">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>190</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="variable" stdset="0">
    <string>SRC03-RF-SSA$(ssa)-REC$(rec):getCurrent</string>
   </property>
   <property name="variableSubstitutions" stdset="0">
    <string>rec=02</string>
   </property>
  </widget>
  <widget class="QELabel" name="qelabel_15">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>170</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="variable" stdset="0">
    <string>SRC03-RF-SSA$(ssa)-REC$(rec):getAcVoltage</string>
   </property>
   <property name="variableSubstitutions" stdset="0">
    <string>rec=02</string>
   </property>
  </widget>
  <widget class="QELabel" name="qelabel_16">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>210</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="variable" stdset="0">
    <string>SRC03-RF-SSA$(ssa)-REC$(rec):getPower</string>
   </property>
   <property name="variableSubstitutions" stdset="0">
    <string>rec=03</string>
   </property>
  </widget>
  <widget class="QELabel" name="qelabel_17">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>190</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="variable" stdset="0">
    <string>SRC03-RF-SSA$(ssa)-REC$(rec):getCurrent</string>
   </property>
   <property name="variableSubstitutions" stdset="0">
    <string>rec=03</string>
   </property>
  </widget>
  <widget class="QELabel" name="qelabel_18">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>170</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="variable" stdset="0">
    <string>SRC03-RF-SSA$(ssa)-REC$(rec):getAcVoltage</string>
   </property>
   <property name="variableSubstitutions" stdset="0">
    <string>rec=03</string>
   </property>
  </widget>
  <widget class="QELabel" name="qelabel_19">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>210</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="variable" stdset="0">
    <string>SRC03-RF-SSA$(ssa)-REC$(rec):getPower</string>
   </property>
   <property name="variableSubstitutions" stdset="0">
    <string>rec=04</string>
   </property>
  </widget>
  <widget class="QELabel" name="qelabel_20">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>190</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="variable" stdset="0">
    <string>SRC03-RF-SSA$(ssa)-REC$(rec):getCurrent</string>
   </property>
   <property name="variableSubstitutions" stdset="0">
    <string>rec=04</string>
   </property>
  </widget>
  <widget class="QELabel" name="qelabel_21">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>170</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="variable" stdset="0">
    <string>SRC03-RF-SSA$(ssa)-REC$(rec):getAcVoltage</string>
   </property>
   <property name="variableSubstitutions" stdset="0">
    <string>rec=04</string>
   </property>
  </widget>
  <widget class="QELabel" name="qelabel_22">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>210</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="variable" stdset="0">
    <string>SRC03-RF-SSA$(ssa)-REC$(rec):getPower</string>
   </property>
   <property name="variableSubstitutions" stdset="0">
    <string>rec=05</string>
   </property>
  </widget>
  <widget class="QELabel" name="qelabel_23">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>190</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="variable" stdset="0">
    <string>SRC03-RF-SSA$(ssa)-REC$(rec):getCurrent</string>
   </property>
   <property name="variableSubstitutions" stdset="0">
    <string>rec=05</string>
   </property>
  </widget>
  <widget class="QELabel" name="qelabel_24">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>170</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="variable" stdset="0">
    <string>SRC03-RF-SSA$(ssa)-REC$(rec):getAcVoltage</string>
   </property>
   <property name="variableSubstitutions" stdset="0">
    <string>rec=05</string>
   </property>
  </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QELabel</class>
   <extends>QLabel</extends>
   <header>QELabel.h</header>
  </customwidget>
  <customwidget>
   <class>QELineEdit</class>
   <extends>QLineEdit</extends>
   <header>QELineEdit.h</header>
  </customwidget>
  <customwidget>
   <class>QEPushButton</class>
   <extends>QPushButton</extends>
   <header>QEPushButton.h</header>
  </customwidget>
  <customwidget>
   <class>QSimpleShape</class>
   <extends>QWidget</extends>
   <header>QSimpleShape.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
